coalib/output/Logging.py

```diff
@@ -1,10 +1,26 @@
 from datetime import datetime
+from collections import defaultdict
 import json
 import io
 import logging
 import logging.config
 
 
+class CounterHandler(logging.Handler):
+    """
+    A logging handler which counts the number of calls
+    for each logging level.
+    """
+    level_called = None
+
+    def __init__(self, *args, **kwargs):
+        super(CounterHandler, self).__init__(*args, **kwargs)
+        CounterHandler.level_called = defaultdict(lambda: 0)
```

Comment:
Hm, so I had to leave the `__init__` in, so the counter would get reset for each run. It would probably work "in the wild" because coala will only run once per import, but for the tests, I see no other way.